tests/test_datastructures.py

```diff
@@ -795,23 +755,15 @@ def test_to_wsgi_list(self):
         h = self.storage_class()
         h.set(u"Key", u"Value")
         for key, value in h.to_wsgi_list():
-            if PY2:
-                strict_eq(key, b"Key")
-                strict_eq(value, b"Value")
-            else:
-                strict_eq(key, u"Key")
-                strict_eq(value, u"Value")
+            strict_eq(key, u"Key")
```

Comment:
Can get rid of at least `u""`, and probably `strict_eq`.

Reply:
Done.